modules/nodes-vsphere-machine-set-scaling-static-ip.adoc

```diff
@@ -0,0 +1,188 @@
+
+[source, terminal]
+----
+oc create -f <ipaddressclaim_filename>
+----
+
+See https://docs.openshift.com/container-platform/4.13/networking/multiple_networks/configuring-additional-network.html#nw-multus-ipam-object_configuring-additional-network 
+====
+
+The following demonstrates an example of an `IPAddressClaim` resource:
+
+[source, yaml]
+----
+kind: IPAddressClaim
+metadata:
+  finalizers:
+  - machine.openshift.io/ip-claim-protection
+  name: cluster-dev-9n5wg-worker-0-m7529-claim-0-0
+  namespace: openshift-machine-api
+spec:
+  poolRef:
+    apiGroup: ipamcontroller.openshift.io
+    kind: IPPool
+    name: static-ci-pool
+status: {}
+----
+
+The machine controller updates the machine with a status of `IPAddressClaimed` to indicate that a static IP address has succesfully bound to the `IPAddressClaim` resource. The machine controller applies the same status to a machine with multiple `IPAddressClaim` resources that each contain a bound static IP address.The machine controller then creates a virtual machine and applies static IP addresses to any nodes listed in the `providerSpec` of a machine's configuration.
+
+The example in the procedure demonstrates the use of controllers for scaling machines in a machine set.
+
+.Prerequisites
+
+* ANY?
+
+.Procedure
+. Configure a machine set by specifying IP pool information in the `network.devices.addressesFromPools` schema of the machine set's YAML file: 
++
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  annotations:
+    machine.openshift.io/memoryMb: "8192"
+    machine.openshift.io/vCPU: "4"
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+  name: <infrastructure_id>-<role>
+  namespace: openshift-machine-api
+spec:
+  replicas: 0
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>
+  template:
+    metadata:
+      labels:
+        ipam: "true"
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>
+    spec:
+      lifecycleHooks: {}
+      metadata: {}
+      providerSpec:
+        value:
+          apiVersion: machine.openshift.io/v1beta1
+          credentialsSecret:
+            name: vsphere-cloud-credentials
+          diskGiB: 120
+          kind: VSphereMachineProviderSpec
+          memoryMiB: 8192
+          metadata: {}
+          network:
+            devices:
+            - addressesFromPools: <1>
+              - group: ipamcontroller.openshift.io
+                name: static-ci-pool
+                resource: IPPool
+              nameservers:
+              - "192.168.204.1" <2>
```

Comment:
SME query: Internal [writer guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#ip-addresses) state that only a certain range of IP addresses should be used in public-facing docs (depends on whether address are private or public). Are the addresses in the example private addresses?

![Screenshot from 2023-07-18 12-06-36](https://github.com/openshift/openshift-docs/assets/57954076/17fcdd63-ed0a-4de7-bcf8-8eda71647d36)

Reply:
These are all private IP addresses.  That said, if they need to be changed per guidelines I have no issue with that.

Reply:
Thanks: `192.168.204.1` falls into the acceptable range according to the above guidance.